src/library/scala/collection/LinearSeqOptimized.scala

```diff
@@ -117,25 +117,25 @@ trait LinearSeqOptimized[+A, +Repr <: LinearSeqOptimized[A, Repr]] extends Linea
   }
 
   override /*TraversableLike*/
-  def foldLeft[B](z: B)(f: (B, A) => B): B = {
+  def foldLeft[B](z: B)(op: (B, A) => B): B = {
```

Comment:
Are you aware of `@deprecatedName`? Sample https://github.com/scala/scala/blob/2.11.x/src/library/scala/concurrent/Future.scala#L134